symbols/arm9.yml

```diff
@@ -1472,6 +1548,21 @@ arm9:
         These variables underpin the various ExplorerScript global variables you can use in the SkyTemple SSB debugger.
         
         type: struct script_var_table
+    - name: KAOMADO_LAYOUTS
+      address:
+        NA: 0x209E014
+      length:
+        NA: 0xC0
+      description: |-
+        All the possible layouts a Kaomado portrait can be placed in by default
+        
+        type: {struct kaomado_layout}[32]
```

Comment:
Remove the curly braces to make it conventional C syntax: `struct kaomado_layout[32]`. I only use the curly brace hack for "inline" struct definitions that aren't in the C headers.